releases/v19.1.10.md

```diff
@@ -0,0 +1,84 @@
+---
+title: What&#39;s New in v19.1.10
+toc: true
+summary: Additions and changes in CockroachDB version v19.1.10 since version v19.1.9
+---
+
+## June 29, 2020
+
+This page lists additions and changes in v19.1.9 since v19.1.8.
+
+- For a comprehensive summary of features in v19.1, see the [v19.1 GA release notes](v19.1.0.html).
+- To upgrade to v19.1, see [Upgrade to CockroachDB v19.1](../v19.1/upgrade-cockroach-version.html)
```

Comment:
This feels odd to me -- Do we recommend upgrading to an older version when a newer one is available?  🤔